### PR TITLE
Pushing the image to GHCR.io

### PR DESCRIPTION
**Pull request by Praveenraj-K**
**Added** 

- Pushing the image to GHCR.io

### CODE REVIEW

**Review by hobbitronics — APPROVED** [review]
Looks good to me!

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -25,12 +25,21 @@ jobs:
...
+            password: ${{ secrets.GITHUB_TOKEN }}
 
       - name: Extract metadata (tags, labels) for Docker
         id: meta
         uses: docker/metadata-action@v5
         with:
-          images: ${{ vars.DOCKER_ORG }}/traefik-https-proxy
+          images: |
+            ${{ vars.DOCKER_ORG }}/traefik-https-proxy
+            ghcr.io/${{ github.repository_owner }}/traefik-https-proxy
```
Comment (hobbitronics) [inline .github/workflows/ci.yml:42]: is it possible to use `${{ github.event.repository.name }}` here? Just wondering which is preferable

Comment (briskt) [reply .github/workflows/ci.yml:42]: It actually should be `ghcr.io/${{ github.repository }}` to simplify.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -45,3 +54,4 @@ jobs:
           push: true
           tags: ${{ steps.meta.outputs.tags }}
           labels: ${{ steps.meta.outputs.labels }}
+
```
Comment (briskt) [inline .github/workflows/ci.yml:57]: @Praveenraj-K also, please set your IDE to add null-termination at the end of the file.

Here's the setting in VSCode:
image

Comment (hobbitronics) [reply .github/workflows/ci.yml:57]: I find the diff confusing. Here I thought he added a new line due to the green. I even asked copilot and it agreed!

Comment (briskt) [reply .github/workflows/ci.yml:57]: I agree, it's a bit strange. But it looks like he added an extra non-terminated blank line. Here's the git diff output after I deleted that blank line in vim.
```
14:26  $ git diff
diff --git a/.github/workflows/ci.yml b/.github/workflows/ci.yml
index b059386..5314d1c 100644
--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -54,4 +54,3 @@ jobs:
           push: true
           tags: ${{ steps.meta.outputs.tags }}
           labels: ${{ steps.meta.outputs.labels }}
-          
\ No newline at end of file
```

Comment (Praveenraj-K) [reply .github/workflows/ci.yml:57]: > I find the diff confusing. Here I thought he added a new line due to the green. I even asked copilot and it agreed!

Thanks for @briskt  updated the setting in the vs code.

**Pull request merged**